### ISSUE TITLE
Hacky solution to a dumb issue

### ISSUE THREAD

**Issue by Johnsmith0508** [Johnsmith0508/cubes#50]
> for reference, see 9326567

due to the async nature of THREE's model loaders, when [clientjs:430](https://github.com/Johnsmith0508/cubes/blob/master/js/client.js#L430) creates the new Item, the model is not always created yet. To combat this issue, a dummy model is made, with only a `.clone()` function, to satisfy the needs of the constructor. This solution is a _terrible_ Idea, for one because it is a hack required for many implementations of the API. It also could cause other issues down the line, due thinking the model exists, when it does not.

**TL;DR**
created dummy model to satisfy a constructor. don't do that
